### PR TITLE
feat: add my info

### PR DESCRIPTION
**Pull request by Hypell**
added info

### CODE REVIEW

File: READMEhyeri.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # 박혜리를 소개합니다. 😎
 
+| 이름 | 학과 | 학번 |
+| :---: | :---: | :---: |
+| 박혜리 | 영어교육과 | 20 |
```
Comment (Yeonghunj) [inline READMEhyeri.md:5]: very nice

File: READMEhyeri.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # 박혜리를 소개합니다. 😎
 
+| 이름 | 학과 | 학번 |
```
Comment (Yeonghunj) [inline READMEhyeri.md:3]: 멋져

File: READMEhyeri.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
...
+| :---: | :---: | :---: |
```
Comment (Yeonghunj) [inline READMEhyeri.md:4]: 잘했어!!

**Review by Yeonghunj — COMMENTED** [review]
ok

File: READMEhyeri.md (hunk 1 above)
Comment (yebin1926) [reply READMEhyeri.md:5]: 짱이다

**Pull request merged**